nuxeo-services/nuxeo-platform-actions-core/src/main/java/org/nuxeo/ecm/platform/actions/ActionContext.java

```diff
@@ -99,6 +99,14 @@
      */
     boolean checkCondition(String expression) throws ELException;
 
+    /**
+     * Eval the given {@code expression} and returns the result casted to the given {@code expectedType}.
+     *
+     * @return The result of the expression evaluation.
```

Comment:
for `@return`/`@param`, no initial uppercase and no terminating period

Reply:
Too bad, I did get that javadoc from `javax.el.ValueExpression`! 😄